### ISSUE TITLE
Why not use the official cross_entropy loss?

### ISSUE THREAD

**Issue by ksnzh** [meetshah1995/pytorch-semseg#113]
Since the official CrossEntropy support 2D image multinomial logistic loss, what the difference between cross_entropy2d of this repo and the official implementation?
